examples/gimbal_device_tester/gimbal_device_tester.cpp

```diff
@@ -270,24 +270,26 @@ class Tester {
             return false;
         }
 
-        std::cout << test_prefix << "Yaw 20 degrees to the right... ";
-        _attitude_data.change_vehicle_attitude([](AttitudeData::VehicleAttitude& vehicle_attitude) {
-            vehicle_attitude.yaw_deg = 20.0f;
-        });
-        std::this_thread::sleep_for(std::chrono::seconds(2));
-        std::cout << "DONE\n";
+        // Let's not yaw for now because it is confusing for a gimbal.
+        // std::cout << test_prefix << "Yaw 20 degrees to the right... ";
+        //_attitude_data.change_vehicle_attitude([](AttitudeData::VehicleAttitude& vehicle_attitude)
+        //{
+        //    vehicle_attitude.yaw_deg = 20.0f;
+        //});
+        // std::this_thread::sleep_for(std::chrono::seconds(25));
+        // std::cout << "DONE\n";
```

Comment:
```suggestion
```

Reply:
Not this time :smile:

Reply:
Do we need to keep this unused code? :thinking:

Reply:
Yes, so that a reader sees how it would work.

Reply:
Or if I use it against SITL I would use it.